vpn/src/main/java/com/duckduckgo/mobile/android/vpn/waitlist/store/AtpWaitlistStateRepository.kt

```diff
@@ -42,9 +43,18 @@ class WaitlistStateRepository @Inject constructor(
         return WaitlistState.NotJoinedQueue
     }
 
-    fun didJoinBeta(): Boolean = dataStore.inviteCode != null
+    override fun joinedAfterCuttingDate(): Boolean {
+        return dataStore.waitlistTimestamp > CUTOFF_DATE
+    }
```

Comment:
This seems very unflexible logic. Why don't we put the cutoff date in the survey JSON file. 
That would make this method something like `didJoinAfterCuttingDate(...): Boolean`.